### PR TITLE
Add support for Electronut Labs Papyr.

### PR DESCRIPTION
**Pull request by ntavish**
Signed-off-by: Tavish Naruka <tavishnaruka@gmail.com>

### CODE REVIEW

File: ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x239A
```
Comment (tannewt) [inline ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk]: Do you have your own VID for USB?

Comment (ntavish) [reply ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk]: Nope, I missed that. What do you suggest I put there?

**Comment by ntavish** [comment]
> Very cool! I have one question but looks good otherwise. Have you tested battery life at all? We haven't done any optimization of CircuitPython yet for it.

Haven't looked at any optimizations yet, but got the display working. I was playing with CircuitPython for the first time today, I'll take a look at it later.
D4OcIgJUUAA4YCD jpg:large

File: ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk (hunk 1 above)
Comment (tannewt) [reply ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk]: Ok, no problem. We'll give you a set of Adafruit PID numbers. So please use `0x003b` for the bootloader, `0x803b` for arduino and `0x803c` for CircuitPython (on the next line.) The VID can remain 0x239A. Thanks for trying CircuitPython!

Comment (ntavish) [reply ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk]: thank you @tannewt! I'll update the PRs.

File: ports/nrf/boards/makerdiary_nrf52840_mdk/mpconfigboard.mk [hunk 2]
```diff
@@ -1,5 +1,5 @@
 USB_VID = 0x239A
-USB_PID = 0x802A
+USB_PID = 0x803C
```
Comment (tannewt) [inline ports/nrf/boards/makerdiary_nrf52840_mdk/mpconfigboard.mk]: Please change the .mk file for your board.

**Pull request merged**